File: 4-Estruturas-e-Bibliotecas/3357-Rico-do-Mate/3357.py
people, bottle, bowl = input().split()
people, bottle, bowl = int(people), float(bottle), float(bowl)
contestants = input().split()
dose = 1
while bottle - bowl*dose > 0:
    dose += 1
dose -= 1
print (contestants[dose%people], round((bottle-bowl*(dose)), 1))

File: 4-Estruturas-e-Bibliotecas/3357-Rico-do-Mate/3357.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int people = scan.nextInt();
        float bottle = scan.nextFloat();
        float bowl = scan.nextFloat();
        scan.nextLine();
        String[] contestants = scan.nextLine().split(" ");
        int dose = (int) (bottle/bowl);
        float result = (float)(bottle-bowl*dose);
        if (result == 0) {
            --dose;
            result = (float)(bottle-bowl*dose);
        }
        System.out.printf("%s %.1f\n", contestants[dose%people], result);
    }
}